types/rest/rest_test.go

```diff
@@ -3,15 +3,21 @@
 package rest
 
 import (
+	"encoding/json"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
+	"strconv"
 	"testing"
 
-	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
 
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
Top level packages must not depend on `x` sub packages.

Reply:
Offf, good catch -- I missed this!

Reply:
We can probably remove this test all together @colin-axner and move it to the LCD tests in Gaia.

Reply:
We don’t need LCD tests for queries as those are covered by the contract tests

Reply:
Am I removing and moving the tests elsewhere or should I refactor the test to use a mock version of account?

Reply:
I like the latter